SU-SWS/decanter
# add su- prefix to Decanter core and component variables, deprecate old names

# READY

# Summary
Add `su-` prefix to all variable names to allow people building on top of Decanter to use the more generic names without collision.

# Needed By (Date)
5.0.2

# Urgency
Minor

# Steps to Test

1. Pull the branch
1. Rebuild the assets
1. Confirm that nothing has changed in any of our styles
1. Rebuild the style guide
1. Note the new variables
1. Note the deprecation warning for the old variables

# Affected Projects or Products
This may cause merge conflicts with other PR's, as most of our files refer to at least one variable.

# Associated Issues and/or People
- #382 
- Also tracking this in https://stanfordits.atlassian.net/browse/THEME-14

@JBCSU 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
